partiql-ast/src/main/java/org/partiql/ast/Identifier.java

```diff
@@ -35,11 +39,49 @@ public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
     }
 
     @NotNull
-    public String getSymbol() {
-        return this.symbol;
+    public List<Part> getQualifier() {
+        return this.qualifier;
     }
 
-    public boolean isDelimited() {
-        return this.delimited;
+    @NotNull
+    public Part getBase() {
+        return this.base;
+    }
```

Comment:
While "getIdentifier()" and "identifier" is not ideal (maybe base is better?) I recommend going with "getIdentifier" only for SPI consistency, but I understand what you are going for

Reply:
Yeah, `base` is too generic -- changed to `identifier`.